clocksilencer.ino

```diff
@@ -75,9 +75,9 @@ void loop() {
     }
   }
   struct tm timeinfo = rtc.getTimeStruct();
-  esp_sleep_enable_timer_wakeup(500000); //in microseconds, but not the full second (1000000)
-  if (bootCount > 2147483645){
-    bootCount=2;// when bootcount approaches to integer max, reset it
+  esp_sleep_enable_timer_wakeup(500000);  //in microseconds, but not the full second (1000000)
+  if (bootCount > 2147483645) {
```

Comment:
I like this :)
when we are close to maximum value of `int` we switch the counter to `2` because it could overflow, right?

On the other hand.. do we need the counter?

It looks like the simple `boolean` variable `ntpIsTimeInitialised` would be enough. Byt this could be done in separate merge request :)

Reply:
Well, we don't need that :)
I get your idea, will try to do that.